tests/vmi_multus_test.go

```diff
@@ -457,14 +460,14 @@ var _ = Describe("Multus", func() {
 					Expect(is_present).To(BeTrue())
 					Expect(ifc.MAC).To(Not(BeZero()))
 				}
-				Expect(interfacesByName["default"].MAC).To(Not(Equal(interfacesByName["ovs"].MAC)))
+				Expect(interfacesByName["default"].MAC).To(Not(Equal(interfacesByName["linux-bridge"].MAC)))
```

Comment:
can we create a constant for `linux-bridge`, something like `defaultInterfaceName`

Reply:
defaultInterfaceName is "default" though (on the left).

i don't want to do any refactoring in this PR, i want to keep it to the point (switch from OVS to linux bridge).